synapse/cryotank.py

```diff
@@ -102,6 +116,8 @@ def puts(self, items):
                 info = {'time': tick, 'count': len(items), 'size': bytesize, 'took': took}
                 curs.put(lkey, s_msgpack.en(info), append=True)
 
+        self.fire('cryotank:puts')
```

Comment:
since we're firing the event, lets include the length of the number of items put into the tank here?

Reply:
ok